extensions/script/tests/CMakeLists.txt

```diff
@@ -49,6 +45,17 @@ FOREACH(testfile ${EXECUTESCRIPT_PYTHON_TESTS})
     createTests("${testfilename}")
     MATH(EXPR EXTENSIONS_TEST_COUNT "${EXTENSIONS_TEST_COUNT}+1")
     add_test(NAME "${testfilename}" COMMAND "${testfilename}" WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+
+    if(EXTENSIONS_TEST_COUNT EQUAL 1)
+        # Copy test resources
+        add_custom_command(
```

Comment:
I think it would be helpful to explain why exactly this happens only on the first test, and not the others. Would you mind adding a comment about that?

Correct me if I'm wrong, but if I understand correctly, the build step of the first test target will copy all of the test resources to the build directory, where it will be accessible to all of the tests.

Reply:
Good point, added comment in f0fa8ce115fe1c892e25d14d87b6bbb26c586f1f